### ISSUE TITLE
Alerts missing/delayed

### ISSUE THREAD

**Issue by cityshrimp** [EvidentSecurity/esp-sdk-python#7]
External accounts created using Python SDK have significantly slower alert ramp-up time than creating it manually via ESP web.  There are also instances where it seems some signatures never run (even after waiting a full day).  Is there something different that Python SDK does in terms of creating new external accounts that would cause this?

**Comment by johnathanludwig** [comment]
No, the logic for creating accounts via the web requests and API requests are the same. As long as your account was created it should be working normally.

I would suggest contacting support@evident.io so that they could research what is happening. It may be something specific to the account that you added. Make sure to provide as much details as you can including the signatures that you aren't seeing results for.
